web_app/dist/title.css

```diff
@@ -0,0 +1,65 @@
+html {
+    background: #93ca68;
+}
+
+h1 {
+    font-size: 100px;
+    font-family: "Raleway Light", Helvetica, Arial, sans-serif;
+    padding: 0;
+    margin: 0;
+}
+
+h2 {
+    font-size: 30px;
+    font-family: "Raleway-Light", Helvetica, Arial, sans-serif;
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    padding: 70px 0;
+    text-align: center;
+    color: white;
+    font-family: sans-serif;
+    vertical-align: middle;
+}
+
+.box {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.15;
+}
+
+.box2 {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    flex-direction: column;
+}
+
+.footer {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    background-color:black;
+    opacity: 0.7;
+    color: white;
+    text-align: center;
+}
+
+@font-face {
```

Comment:
Be consistent with where this goes in `.css` files, I think it should be at the top like the other one